daml/Marketplace/Custody/Model.daml

```diff
@@ -145,3 +174,10 @@ createNewVersionOfDescription description postAssetVersion serializedRemainingCl
     claims = serializedRemainingClaims
  where
   adjustedId = description.assetId with version = postAssetVersion
+
+createTradeId : Id -> Int -> Text -> Id
+createTradeId assetId transferNumber uniqueId =
+  Id with
+    signatories = assetId.signatories
+    label = assetId.label <> " " <> show transferNumber <> "th transfer, ID: " <> uniqueId
```

Comment:
what is `th transfer` mean ?

Reply:
"4th transfer", there can be many transfers / payouts when process/lifecycle (the choice) is called.

Reply:
ok, that makes sense